File: Completed/Go/485.go
func findMaxConsecutiveOnes(nums []int) int {
	result := 0
	count := 0
	for _, n := range nums {
		if n == 0 {
			count = 0
		} else {
			count += 1
		}

		if count > result {
			result = count
		}
	}

	return result
}

File: Completed/python3/485.py
class Solution:
    def findMaxConsecutiveOnes(self, nums: List[int]) -> int:
      result = 0
      tmp = 0
      for v in nums:
        if v == 1:
          tmp+=1
        else:
          tmp=0
          
        result = max(result,tmp)
      return result
        
